front/src/components/UI/Tabs/EditTabs/EditTabs.tsx

```diff
@@ -0,0 +1,46 @@
+import { useState } from 'react';
+import { PackageData } from '../../../App/Help/Help';
+import { EditTabsButtonStyled, EditTabsInputStyled, EditTabsInputWideStyled} from './EditTabs.styled';
+
+const EditTabs = (props: any) => {
```

Comment:
zamiast any napisz i użyj proszę interfejsu z propsami. Na takiej samej zasadzie jak to jest zrealizowane np. w komponencie Button.

any "wyłącza" w danym miejscu TypeScripta i tracimy podpowiadanie składni i sprawdzanie błędów